src/fluree/db/peer/http_api.clj

```diff
@@ -642,7 +642,7 @@
                                   acc* (into acc [{(keyword k) (count v)}])]
                               (recur r acc*))
                             acc))
-              nw-data   (remove-deep [:private-key] networks)
+              nw-data   (-> (remove-deep [:private-key] networks) vector)
```

Comment:
This would be better written as either of the following:

`(->> networks (remove-deep [:private-key]) vector)`

or

`(vector (remove-deep [:private-key] networks))`